e2e/README.md

```diff
@@ -0,0 +1,31 @@
+# E2E tests
+
+
+## Directory structure
+
+- `test-app-v10`: example app using Angular v10 with basic setup with `jest` and `jest-preset-angular`.
+- `test-app-v9`: example app using Angular v9 with basic setup with `jest` and `jest-preset-angular`.
+
+
+## Running tests
+
+You run the tests on example apps with `yarn test:e2e` (or `npm run test:e2e)`. In `scripts/lib/paths.js`,
+`projectsToRun` must be modified if a new project is added to `e2e`.
+
+What will happen in order is as following:
+
+1. A bundle (we'll call it `[bundle]`) will be created for `jest-preset-angular` using `npm pack` (`yarn pack` is buggy).
+
+    The `prepare` script will be run, so `clean` and `build` (that is why e2e tests are launched before the others, since it's building as part of the process)
+ 
+2. `npm ci` and `npm install --save-dev [bundle]` will be run in each project directory in `e2e` directory.
+3. each test suite in each example project are run according to `jest` config of that project.
+
+
+## Why
+
+E2E tests being in a temporary folder of the operating system ensure none of the parents would be containing `node_modules`.
```

Comment:
I don't see any reference to creating a temporary dir in the code. I can only see `chdir` to `e2e/<project>` and running commands there. In such case, I'd highly suggest using Jest's "projects" feature:
https://jestjs.io/docs/en/configuration#projects-arraystring--projectconfig and use Yarn workspaces to minimize the network payload when downloading all dependencies of the project and e2e tests. This may be challenging to set up, but faster builds, less maintenance and less network payloads are really nice to have

Reply:
If there's no Yarn workspace in the project now, I'd consider adding it later, after this PR is merged, because now with e2e it makes sense to use it

Reply:
ok let's do it in a separate PR to setup yarn workspace + jest "project"

Reply:
The issue is that we have different behavior of the preset for a Angular v9 and v10 App, so we want to test both. If we use yarn workspaces and only install the dependencies once, we should not be able to install different Angular versions. Correct me if I'm wrong.

Reply:
Ok actually I think this will work with yarn workspaces, but we will see in the follow-up PR.